pkg/controller/kubedirectorcluster/members.go

```diff
@@ -284,7 +284,27 @@ func handleReadyMembers(
 				return
 			}
 
-			memberVersion := *m.StateDetail.LastConnectionVersion
+			// memberVersion is the connectionVersion of the member.
+			// If it is nil, we set it to connectionsVersion - 1. This is to call startscript with --reconnect and then equalize connectionsVersion and memberVersion
+			// If it is not nil, we set it to the member's connectionVersion
+			var memberVersion int64
+
+			if m.StateDetail.LastConnectionVersion != nil {
+				memberVersion = *m.StateDetail.LastConnectionVersion
+
+			} else {
+
+				shared.LogInfof(
+					reqLogger,
+					cr,
+					"",
```

Comment:
Let's use the shared.EventReasonNoEvent constant here, rather than hardcoding its value.

Reply:
Done